marketplace-service/src/main/java/com/axonivy/market/util/VersionUtils.java

```diff
@@ -126,7 +125,7 @@ public static String getOldestVersion(List<GHTag> tags) {
         return result;
     }
     public static List<String> getReleaseTagsFromProduct(Product product) {
-        if (Objects.isNull(product)) {
+        if (Objects.isNull(product) || Objects.isNull(product.getReleasedVersions())) {
```

Comment:
Objects.isNull(product.getReleasedVersions()) this one is collection, so we should use collectiontuil is empty